### ISSUE TITLE
Testing: Backend

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#34]
## User story
1. As a {HR employee}
2. I want / need {a well functioning frontend without bugs}
3. So that {I can efficiently and effectively interact with the system and trust in it}

## Acceptance criteria
* Create a general concept for testing of the backend functionalities: How can a high quality of the components be assured?
* How can the connection to the Lissi backend be tested?
* Think of first testing scenarios. Be able to present them, know broadly how they can be implemented. No need for actual implementation yet.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
@veitmo 
Same issues as in #33. See it for the discussion.

**Comment by Idontker** [comment]
@jackDS008 Sag mal Bescheid, wenn du fertig bist. Ich würde es dann nutzen um meinen Task für die Woche zu testen :)

**Comment by jackDS008** [comment]
@Idontker Habe morgen um 10:00 Uhr ein Meeting mit @Jean28518. Danach sollte es fertig sein

**Comment by jackDS008** [comment]
@Idontker Bin fertig. Willst du auch gleich reviewen? [Doku](https://github.com/amosproj/amos2022ss04-digital-identity/blob/main/Documentation/dev/testing/backend/TestingBackend.md) und [Pull request](https://github.com/amosproj/amos2022ss04-digital-identity/pull/85)
